gui/default/syncthing/settings/advancedSettingsModalView.html

```diff
@@ -167,6 +167,26 @@ <h4 class="panel-title" tabindex="0" translate>Default Device</h4>
                 </form>
               </div>
             </div>
+            <div class="panel panel-default">
+              <div class="panel-heading" role="tab" id="advancedDefaultIgnoresHeading" data-toggle="collapse" data-parent="#advancedDefaults" href="#advancedDefaultIgnores" aria-expanded="false" aria-controls="advancedDefaultIgnores" style="cursor: pointer;">
+                <h4 class="panel-title" tabindex="0" translate>Default Ignore Patterns</h4>
+              </div>
+              <div id="advancedDefaultIgnores" class="panel-collapse collapse" role="tabpanel" aria-labelledby="advancedDefaultIgnoresHeading">
+                <form class="form-horizontal" role="form">
+                  <div ng-repeat="(key, value) in advancedConfig.defaults.ignores" ng-if="inputTypeFor(key, value) != 'skip'" class="form-group">
```

Comment:
Is this for future extensibility? Afaik currently there's just a single `lines` key here, i.e. no looping and no `inputTypeFor` conditions below will apply.

Reply:
Yes, I added it so we have the same generic mechanism for future additions as above without touching this code again. I could imagine some boolean options for when the patterns should apply for example.